src/main/resources/db/migration/postgres/V1_320_1__copyCaseNumbersToMediaLinkedCase.sql

```diff
@@ -0,0 +1,20 @@
+DO
+$do$
+DECLARE
+   case_id   int;
+   courthouse_name   text;
+	temprow record;
+BEGIN
+FOR temprow IN
+        select med_id, ctr_id, UNNEST(case_number) case_number from darts.media where case_number is not null order by 1,2
+    LOOP
+		select cc.cas_id into case_id from darts.court_case cc, darts.courtroom cr where cr.cth_id = cc.cth_id and cc.case_number = temprow.case_number and cr.ctr_id = temprow.ctr_id;
+		if not found then
+			select ch.courthouse_name into courthouse_name from darts.courtroom cr, darts.courthouse ch where cr.cth_id = ch.cth_id and cr.ctr_id= temprow.ctr_id;
+        	INSERT INTO darts.media_linked_case (mlc_id,med_id, courthouse_name, case_number) VALUES (nextval('mlc_seq'),temprow.med_id,courthouse_name, temprow.case_number);
```

Comment:
I notice on new add audios through the api we always create and associated a case src/main/java/uk/gov/hmcts/darts/audio/component/impl/AddAudioRequestMapperImpl.java. Is this a requirement for the migrated data also?

Reply:
this script wont be run for migrated data, this is purely for whatever info the testers had in the test database to try to copy it over, but if it isn't copied, its not a big deal, its more of an audit table. no cases should be created by this script